app/app.iml

```diff
@@ -64,14 +64,6 @@
       <sourceFolder url="file://$MODULE_DIR$/src/main/jni" isTestSource="false" />
```

Comment:
@kisaacs By convention, we refrain from committing auto generated stub (files) like app/app.iml